packages/commons/src/config/index.ts

```diff
@@ -11,4 +11,5 @@ const Config = z
     logLevel: c.LOG_LEVEL,
   }));
 
+export type CommonConfig = z.infer<typeof Config>;
```

Comment:
```suggestion
export type Config = z.infer<typeof Config>;
```
(I'd suggest to rename when importing, like for the object `config` below)